### PR TITLE
LPS-12345

### PR DESCRIPTION
**Pull request by Youness-ELMO**

### CODE REVIEW

**Review by manoelcyreno — CHANGES_REQUESTED** [review]
@Youness-ELMO ,
In general is better than the previous PR that you sent.

1. Fix the problems mentioned.
2. Pay attention to the Given, When, Then, cuz in some cases you are writing GIVEN, when, then ... review it, to it follow the same pattern.
3. Review the method names, some of them present a Pattern error.
4. Review the method names... "CanDeleteBlog" , what it mean, that you will delete with success or failure? will validate with a right values or invalid values?

File: portal-web/test/functional/com/liferay/portalweb/tests/myTestPoshi/macro/TestTraining.macro [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+	macro VerifyTheResultForm {
+		FormsAdminNavigator.gotoViewEntries();
+
+		LexiconEntry.viewEntryName(rowEntry = "Youness EL MODAHHAB");
+
+		LexiconEntry.viewEntryContent(
+			rowAny = "Youness EL MODAHHAB",
+			rowEntry = "Youness EL MODAHHAB");
+	}
+
```
[the remark this answers is not in the document]
Comment (Youness-ELMO) [reply portal-web/test/functional/com/liferay/portalweb/tests/myTestPoshi/macro/TestTraining.macro]: Done

File: portal-web/test/functional/com/liferay/portalweb/tests/myTestPoshi/macro/TestTraining.macro [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+		Type.sendKeys(
+			locator1 = "FileTest#INPUT_HEIGHT",
+			value1 = 1920);
+	}
+
+	macro VerifyCONF {
+		AssertTextEquals.assertText(
+			locator1 = "FileTest#CHECK_CONF",
+			value1 = ${item});
+	}
```
Comment (manoelcyreno) [inline portal-web/test/functional/com/liferay/portalweb/tests/myTestPoshi/macro/TestTraining.macro:129]: this method is not used

File: portal-web/test/functional/com/liferay/portalweb/tests/myTestPoshi/macro/TestTraining.macro [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+	macro ClickControlPanel {
+		Click(locator1 = "FileTest#CLICK_CTP");
+	}
+
+	macro CreateBlogs {
+		Blogs.addEntry(
+			entryContent = ${entryContent},
+			entryTitle = ${entryTitle});
+	}
+
```
Comment (manoelcyreno) [inline portal-web/test/functional/com/liferay/portalweb/tests/myTestPoshi/macro/TestTraining.macro:40]: this macro is not necessary, you can add this code directly on the testcase file

File: portal-web/test/functional/com/liferay/portalweb/tests/myTestPoshi/macro/TestTraining.macro [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+		Form.editElementSetFieldLabel(fieldValue = ${fieldValue2});
+
+		Form.save();
+
+		Form.publishForm();
+	}
+
+	macro ClickControlPanel {
+		Click(locator1 = "FileTest#CLICK_CTP");
+	}
```
Comment (manoelcyreno) [inline portal-web/test/functional/com/liferay/portalweb/tests/myTestPoshi/macro/TestTraining.macro:33]: this macro is not necessary, you can add this code directly on the testcase file

**Comment by manoelcyreno** [comment]
looks good, with small changes

**Comment by manoelcyreno** [comment]
ci:close

**Comment by Youness-ELMO** [comment]
ci:test

**Pull request closed without merging**